src/services/event.js

```diff
@@ -273,11 +277,83 @@ const makeMatchesFilters = (filters) => (event) => {
   return false;
 };
 
+/**
+ * Splits up multiple occurrences of events into individual events
+ * and re-sorts the array
+ *
+ * @param {Object[]} events the array of events to process
+ * @returns {Object[]} the flattened array of events after splitting multiple occurrences
+*/
+const processMultipleOccurences = (events) => {
+
+  var splitEvents = [];
+
+  for (let i = 0; i < events.length; i++) {
+
+    let thisEvent = events[i];
+
+    // if the event has more than 1 occurrences, loop over them and create new events for each; otherwise add the event itself
+    if (thisEvent.Occurrences && thisEvent.Occurrences.length > 1) {
+
+      for (let j = 0; thisEvent.Occurrences && j < thisEvent.Occurrences.length; j++) {
+
+        const thisOccurrence = thisEvent.Occurrences[j];
+        const thisOccurrenceStartDateTime = DateTime.fromISO(thisOccurrence.StartDate).toJSDate();
+        const thisOccurrenceEndDateTime = DateTime.fromISO(thisOccurrence.EndDate).toJSDate();
+        const thisOccurrenceDate = moment(thisOccurrenceStartDateTime).format('MMMM D, YYYY');
+        const thisOccurrenceTimeRange = ("" + moment(thisOccurrenceStartDateTime).format('h:mm A') + " - " + moment(thisOccurrenceEndDateTime).format('h:mm A'));
+        const allOccurrences = thisEvent.Occurrences.map((occurrence) => {return {start: occurrence.StartDate,
```

Comment:
Do we need `allOccurrences` on the objects output from this function? Surely, once we split events into their occurrences, there's no need to store all the occurrences on every occurrence?

Reply:
I did not intend to use it now, I thought that it may be useful to have in the future if we want to reference the other instances of this event from it.

Reply:
I guess whoever does that can just do that work themselves 👍